apps/enc.c

```diff
@@ -99,6 +101,8 @@ int enc_main(int argc, char **argv)
     int ret = 1, inl, nopad = 0;
     unsigned char key[EVP_MAX_KEY_LENGTH], iv[EVP_MAX_IV_LENGTH];
     unsigned char *buff = NULL, salt[PKCS5_SALT_LEN];
+    int pbkdf2 = 0;
+    int iter = 1;
```

Comment:
Yes, it may be better and less confusing. I was thinking that if the user want PBKDF2 with just 1 iteration count, -iter 1 is enough...
What do think ?